translations/strings.yaml

```diff
@@ -337,23 +337,55 @@ telemetryPolicyView:
   description: We strive to provide you with choices and collect only the data we need to improve Mozilla VPN. Sharing this data with Mozilla is optional.
   question: Allow Mozilla VPN to send technical, interaction, and campaign and referral data to Mozilla?
 
+genericError:
+  unexpected: Unexpected error
 
+# Connection Info Screen
 connectionInfo:
   labelBitps:
-    value: bit/s
+    value: bits
     comment: Bits per second
   labelKbitps:
-    value: kbit/s
+    value: kbits
     comment: Kilobits per second
   labelMbitps:
-    value: Mbit/s
+    value: Mbits
     comment: Megabits per second
   labelGbitps:
-    value: Gbit/s
+    value: Gbits
     comment: Gigabits per second
   labelTbitps:
-    value: Tbit/s
+    value: Tbits
     comment: Terabits per second
+  labelPing:
+    value: Ping
+    comment: Latency of the connection
+  unitPing:
+    value: ms
+    commnent: milliseconds
+  loadingIndicatorLabel:
+    value: Testing speed
+    comment: Shown to the user while waiting for the connection speed test results
+  errorMessage: Looks like there was an unexpected error while trying to connect. Please try again or check your connection.
+  # Speed list header for medium and fast speeds
+  listHeaderDefault:
+    value: At your current speed, here's what your device is optimized for
```

Comment:
Use typographic apostrophes `’`. This is also pointed out by the test failure (linter)

Reply:
Added the apostrophes. I was planning to add the colon when using the string. If it adds necessary context for translations I’m happy to add it.